components/events/FollowCalendar.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <div class="event-page__section">
```

Comment:
@memolina2323 - one thing we try to do is keep the classes to their own component. In other words, since this is the `FollowCalendar` component, these class patterns should follow something like:

```vue
<template>
  <div class="follow-calendar-section">
    <h3>Your title</h3>
    <p class="follow-calendar-section__description">Your description</p>
  </div>
</template>
```

That's _typically_ how we do it... 

you might find that you'll need the styles that come w/ the `.event-page__section` can add the styles within this `.vue` file.

Or, when you use the component, you might be able to do something like

```vue
<FollowCalendar class="event-page__section"/>` when you use the component.
```